docs/content/latest/explore/indexes-constraints/indexes-1.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Indexes
+linkTitle: Indexes
+description: Using indexes
+headcontent: Indexes
+image: /images/section_icons/secure/create-roles.png
+menu:
+  latest:
+    identifier: indexes-constraints-indexes-1
+    parent: explore-indexes-constraints
+    weight: 300
+isTocNested: true
+showAsideToc: true
+---
+
+The use of indexes can enhance database performance by enabling the database server to find rows faster. 
+
+YSQL allows you to create, drop, and list indexes, as well as use indexes on expressions.
+
+## Creating Indexes
+
+You create indexes in YSQL using the `CREATE INDEX` statement that has the following syntax:
+
+```
+CREATE INDEX index_name ON table_name(column_list);
+```
+
+*column_list* represents a column or a comma-separated list of several columns to be stored in the index. An index created for more than one column is called a composite index.
+
+You can also create a functional-based index, in which case you would replace *column_list* with an expression. For more information, see [Using Indexes on Expressions](#using-indexes-on-expressions).
```

Comment:
>in which case you would replace *column_list* with an expression

I think any of the elements in the column list can be an expression, not just the whole list.
For instance, this is perfectly fine (though definitely contrived).
```
yugabyte=# create index on t((v * 2), (v + 1), (v + 5));
CREATE INDEX
```

Postgres defines it simply as:

```
{ column_name | ( expression ) }
```
which means a repeated sequence (list) of either column_name or (parenthesis-separated) expression.
But I don't know how they summarize it in text.

Reply:
Changed as follows:

You can also create a functional-based index, in which case you would replace any element of *column_list* with an expression. For more information, see [Using Indexes on Expressions](#using-indexes-on-expressions).